packages/lib/sdk/src/types/virtuals.d.ts

```diff
@@ -0,0 +1,123 @@
+declare namespace Evidence {
+	interface ConfigLayoutOptions {}
+}
+declare module '$evidence/config' {
+	type schema = import('zod').infer<
+		typeof import('../configuration/schemas/config.schema.js').EvidenceConfigSchema
+	>;
+
+	const cfg: Exclude<schema, 'layout'> & {
+		layout: { name: string; options: Evidence.ConfigLayoutOptions };
+	};
+	export default cfg;
+}
+
+declare module '$evidence/build' {
+	export const BUILD_ID: string;
+	export const BUILD_DATE: Date;
+}
+
+declare module '$evidence/bootstrapContexts' {
+	const fn: () => void;
+	export default fn;
+
+	export const asObj: () => Record<string, string>;
+}
+
+declare module '$evidence/contextKeys' {
+	export const QUERIES_CONTEXT_KEY = '__EVIDENCE_CONTEXTUAL_QUERIES';
+	export const ALL_QUERIES_CONTEXT_KEY = '__EVIDENCE_ALL_QUERIES';
+	export const ALL_QUERIES_MUTS_CONTEXT_KEY = '__EVIDENCE_ALL_QUERIES_MUTS';
+}
+
+declare module '$evidence/initUsql' {
+	const initUsqlPromise: Promise<void>;
+	export default initUsqlPromise;
+}
+
+declare module '$evidence/ssrHook.svelte.js' {
```

Comment:
I really don't like the `.svelte.js` prefix when it has nothing to do with runes